.gitignore

```diff
@@ -0,0 +1,3 @@
+/playground
+/mentorship-playground
+/mentorship
```

Comment:
Could this perhaps be just one of these lines? If I recall correctly then we shouldn't need more than one of these directories (your choice)